### PR TITLE
Metabase Docs

### PR DESCRIPTION
**Pull request by nablabits**
Hi MindsDB mates, this PR addresses this issue: https://github.com/mindsdb/mindsdb/issues/2234

## Please describe what changes you made, in as much detail as possible
- Added the docs to set up a connection with Metabase
- Sorted by name left hand side navigation so it will be more intuitive
- Fixed a couple of ports

### CODE REVIEW

File: docs/mindsdb-docs/docs/connect/dbt.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ To integrate your predictions into your DBT workflow, use the dbt-mindsdb adapte
           type: mindsdb
           database: 'mindsdb'
           host: 'cloud.mindsdb.com'
-          port: 47335
+          port: 3306
```
Comment (nablabits) [inline docs/mindsdb-docs/docs/connect/dbt.md:47]: While testing the connection, I came across these guys (this one, and the one in `mysql-client`) that, at least for me, are not working. Happy to drop them from the PR if they work for everyone else.

File: docs/mindsdb-docs/docs/connect/mysql-client.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ You can either connect locally or to a MindsDB Cloud instance, depending on the
 === "MindsDB Cloud"
 
     ```bash
-    mysql -h cloud.mindsdb.com --port 3307 -u [mindsdb_cloud_email] -p [mindsdb_cloud_password]
+    mysql -h cloud.mindsdb.com --port 3306 -u [mindsdb_cloud_email] -p [mindsdb_cloud_password]
```
Comment (nablabits) [inline docs/mindsdb-docs/docs/connect/mysql-client.md:22]: Refer to [this comment](https://github.com/mindsdb/mindsdb/pull/2745/files#r950804920)

File: docs/mindsdb-docs/mkdocs.yml [hunk 3]
```diff
@@ -83,13 +83,14 @@ nav:
     - Vultr Cloud: setup/vultr.md
     - Civo Cloud: setup/civo.md
   - Connect to MindsDB:
-      - MindsDB SQL Editor: connect/mindsdb_editor.md
-      - MySQL CLI: connect/mysql-client.md
       - DBeaver: connect/dbeaver.md
+      - DBT: connect/dbt.md
```
Comment (nablabits) [inline docs/mindsdb-docs/mkdocs.yml:87]: I've sorted by name the entries here as for me feels more natural

**Comment by nablabits** [comment]
I have read the CLA Document and I hereby sign the CLA

File: docs/mindsdb-docs/docs/connect/dbt.md (hunk 1 above)
Comment (ZoranPandovski) [reply docs/mindsdb-docs/docs/connect/dbt.md:47]: Thanks, this is correct `47335` is for local installation

File: docs/mindsdb-docs/mkdocs.yml (hunk 3 above)
Comment (ZoranPandovski) [reply docs/mindsdb-docs/mkdocs.yml:87]: Thanks

**Review by ZoranPandovski — APPROVED** [review]

**Comment by ZoranPandovski** [comment]
recheckcla

**Pull request merged**